src/tests/unit/tests/DetailsView/components/adhoc-tab-stops-test-view.test.tsx

```diff
@@ -2,24 +2,125 @@
 // Licensed under the MIT License.
 
 import { DisplayableVisualizationTypeData } from 'common/types/displayable-visualization-type-data';
+import { FeatureFlagStoreData } from 'common/types/store-data/feature-flag-store-data';
+import { VisualizationScanResultData } from 'common/types/store-data/visualization-scan-result-data';
+import {
+    ScanData,
+    TestsEnabledState,
+    VisualizationStoreData,
+} from 'common/types/store-data/visualization-store-data';
+import { VisualizationType } from 'common/types/visualization-type';
 import {
     AdhocTabStopsTestView,
+    AdhocTabStopsTestViewDeps,
     AdhocTabStopsTestViewProps,
 } from 'DetailsView/components/adhoc-tab-stops-test-view';
+import { DetailsViewToggleClickHandlerFactory } from 'DetailsView/handlers/details-view-toggle-click-handler-factory';
 import { shallow } from 'enzyme';
 import * as React from 'react';
+import { IMock, Mock, MockBehavior } from 'typemoq';
+import { ContentReference } from 'views/content/content-page';
 
 describe('AdhocTabStopsTestView', () => {
-    const props = {
-        configuration: {
-            displayableData: {
-                title: 'test title',
-            } as DisplayableVisualizationTypeData,
-        },
-    } as AdhocTabStopsTestViewProps;
-
-    it('renders with content', () => {
-        const rendered = shallow(<AdhocTabStopsTestView {...props} />);
-        expect(rendered.getElement()).toMatchSnapshot();
+    let props: AdhocTabStopsTestViewProps;
+    let getStoreDataMock: IMock<(data: TestsEnabledState) => ScanData>;
+    let clickHandlerFactoryMock: IMock<DetailsViewToggleClickHandlerFactory>;
+    let displayableDataStub: DisplayableVisualizationTypeData;
+    let scanDataStub: ScanData;
+    let clickHandlerStub: (event: any) => void;
+    let visualizationStoreDataStub: VisualizationStoreData;
+    let selectedTest: VisualizationType;
+    let featureFlagStoreDataStub: FeatureFlagStoreData;
+    let visualizationScanResultData: VisualizationScanResultData;
+    // const props = {
```

Comment:
Looks like this may have been a typo/forgotten, these outdated props can be removed.